assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/TicketPriceCalculatorForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FieldArray } from 'react-final-form-arrays';
+import { H2, HTMLTable } from '@blueprintjs/core';
+
+import TicketPriceModifierRowIterator from './TicketPriceModifierRowIterator';
+import TicketPriceTotalRow from './TicketPriceTotalRow';
+
+// just temporary
+import styles from './inlineStyles';
+
+const TicketPriceCalculatorForm = ({ form, values: { ticket } }): React.ReactNode => {
+	const reverseCalculate = !!ticket.reverseCalculate;
+	const toggleCalcDir = () => form.mutators.toggleCalcDir('ticket.reverseCalculate');
+	const ticketName = ticket.name ? ` for Ticket: ${ticket.name} ` : '';
+	return (
+		<>
+			<H2 style={styles.hdr}>{`Ticket Price Calculator${ticketName}`}</H2>
+			<div style={styles.div}>
+				<HTMLTable interactive striped style={styles.table}>
```

Comment:
The `tailwindcss` that @manzoorwanijk had mentioned has a responsive variant of the html table: https://tailwindcss.com/docs/table-layout/#responsive-and-pseudo-class-variants

Reply:
kewl will have to look into that to see what those do.
I created a responsive table with built in drag n drop support in EDTR v1: assets\ZZZ\components\ui\responsive-table
that we well port over to EDTR v2 when the time comes

The table from blueprint also has drag n drop support but is not responsive.